### ISSUE TITLE
about how to train

### ISSUE THREAD

**Issue by sanwei111** [MCG-NJU/TDN#65]
hello,i want 2 train my own model through your pretrained model,how to put the video and the label?

**Comment by yztongzhan** [comment]
1. Generate annotations needed for dataloader ("<path_to_video> <video_class>" in annotations) The annotation usually includes train.txt and val.txt. The format of *.txt file is like:
        ```
        dataset_root/video_1.mp4  label_1
        dataset_root/video_2.mp4  label_2
        dataset_root/video_3.mp4  label_3
        ...
        dataset_root/video_N.mp4  label_N
        ```
   2. Add the information to `ops/dataset_configs.py`.
   3. Modify this [function](https://github.com/MCG-NJU/TDN/blob/a7c7e9bcd109d8256cd27a11505bd6e00de39ce0/ops/dataset_config.py#L102) and define the `filename_categories = #num_of_class`.
